Add extra instructions on how to create a Twitter app and create Tokens (#682)
tweepy/tweepy: docs/auth_tutorial.rst

Tweepy tries to make OAuth as painless as possible for you. To begin
the process we need to register our client application with

> Twitter. To create a new application you will need to go to 

Reviewer: ```suggestion
Twitter. To create a new application, go to the
```

> `Twitter App page <https://apps.twitter.com>`_ and click on "Create a

Reviewer: ```suggestion
`Twitter Apps page <https://developer.twitter.com/en/apps>`_ and click on "Create an
```

> new app". Follow the instruction and agree with the terms and

Reviewer: ```suggestion
app". Follow the instructions and agree to the terms and
```

> conditions. Now that you have created your application go to "Keys and 

Reviewer: ```suggestion
conditions. Now that you have created your application, go to "Keys and 
```

> Access Tokens". This is where you will find your Consumer Key 

Reviewer: ```suggestion
tokens". This is where you will find your Consumer Key 
```

> (consumer_token), Consumer Secret (consumer_secret).

Reviewer: ```suggestion
(consumer_token) and Consumer Secret (consumer_secret).
```

> Keep these two handy, you'll need them.

Reviewer: ```suggestion
Keep these two handy; you'll need them.
```

   except tweepy.TweepError:
       print 'Error! Failed to get access token.'
   

> Alternatively, to create the tokens you can go on your app page,

Reviewer: ```suggestion
Alternatively, you can go to your app page to create the token and secret:
```

> click on Keys and Access Tokens, then scroll down the page and

Reviewer: ```suggestion
click on "Keys and tokens"; then
```

> click on "Create my access Tokens". You will now see your Access

Reviewer: ```suggestion
click on "Create" under "Access token & access token secret". You will now see your Access
```
